frontend/packages/common/src/intl/locales/en/system.json

```
@@ -23,7 +23,8 @@
   "label.coldchain-confirm-ok-alerts": "Send temperature OK confirmation",
   "label.coldchain-high-temp-alerts": "Send high temperature alerts (Limits are based on your mSupply configuration)",
   "label.coldchain-low-temp-alerts": "Send low temperature alerts (Limits are based on your mSupply configuration)",
```

```suggestion
  "label.coldchain-high-temp-alerts": "Send high temperature alerts",
  "label.coldchain-low-temp-alerts": "Send low temperature alerts",
```
This was doubling up with the `i` information

---

I forgot to fix them!